src/h2load.cc

```diff
@@ -1242,12 +1242,14 @@ int Client::connection_made() {
 
 int Client::on_read(const uint8_t *data, size_t len) {
   auto rv = session->on_read(data, len);
+  if (worker->current_phase == Phase::MAIN_DURATION) {
+    if ( len > 0 ) {
```

Comment:
I think this is not needed.  The previous code does not have this condition.

Reply:
Maybe not, but I wasn't sure if it would be possible for `len` to ever be a negative number. 
So, I was trying to avoid the possibility of bytes being deducted.

Reply:
len is of type size_t that is unsigned integer, never become negative.

Reply:
Oops. Removed the unnecessary check. Thanks.